server/models/Order.ts

```diff
@@ -35,6 +35,7 @@ const OrderModel = z.object({
   curfewConditions: CurfewConditionsModel.nullable().optional(),
   curfewTimeTable: CurfewTimetableModel.optional(),
   interestedParties: InterestedPartiesModel.nullable(),
+  isValid: z.boolean().optional(),
```

Comment:
I wonder whether this should default to false?

```suggestion
  isValid: z.boolean().optional().default('false'),
```